root/static/scripts/edit/components/UrlRelationshipCreditFieldset.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * @flow strict
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import FieldErrors from './FieldErrors.js';
+import FormRowText from './FormRowText.js';
+
+export type ActionT = {+credit: string, +type: 'update-relationship-credit'};
+
+type PropsT = {
+  +dispatch: (ActionT) => void,
+  +field: FieldT<string | null>,
+};
+
+export type StateT = DatePeriodFieldT;
+
+const UrlRelationshipCreditFieldset = ({
+  dispatch,
+  field,
+}: PropsT): React$Element<'fieldset'> => {
+  function handleCreditChange(
+    event: SyntheticKeyboardEvent<HTMLInputElement>,
+  ) {
+    dispatch({
+      credit: event.currentTarget.value,
+      type: 'update-relationship-credit',
+    });
+  }
+
+  return (
+    <fieldset>
+      <legend>{l('Credit')}</legend>
+      <FormRowText
```

Comment:
Should we add some short help text here to indicate when to use this field? It looks a bit sparse with just "Credit." I guess we could just link to https://wiki.musicbrainz.org/Relationships

Reply:
It says "Credited to:" just by the field, but not sure if that's better. To be honest, I'm not sure the one line in the doc page about credits is a lot better either...